install/kubernetes/cilium/values.yaml

```diff
@@ -84,7 +84,7 @@ agent: true
 name: cilium
 
 # -- Roll out cilium agent pods automatically when configmap is updated.
-rollOutCiliumPods: false
+rollOutCiliumPods: true
```

Comment:
turning this flag on will help with config map changes only _from helm_, but not for manual edit AFAIK.

Reply:
That's fair. I'm pretty much assuming that these charts would be the source of any install, so newer installs would have this on by default and hence even if you changed something manually later then it would be taking effect. It's also not uncommon to use these charts to initially generate the templates to apply (rather than e.g. `helm install` or `cilium install` directly).